### PR TITLE
fix: add iPhone 12 Mini

### PR DESCRIPTION
**Pull request by nswbmw**

### CODE REVIEW

**Comment by pavelfeldman** [comment]
The swapped width / height is on purpose, check it using a nearby iPhone.

**Comment by nswbmw** [comment]
@pavelfeldman ok, just add iPhone 12 Mini

**Comment by pavelfeldman** [comment]
Bots are unhappy, run npm run build?

**Comment by mxschmitt** [comment]
@nswbmw can you revert the changes in the readme? Bots should trigger automatically.

File: packages/playwright-core/src/server/deviceDescriptorsSource.json [hunk 1]
```diff
@@ -663,6 +663,36 @@
     "hasTouch": true,
     "defaultBrowserType": "webkit"
   },
+  "iPhone 12 Mini": {
+    "userAgent": "Mozilla/5.0 (iPhone; CPU iPhone OS 14_4 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/15.4 Mobile/15E148 Safari/604.1",
+    "screen": {
+      "width": 375,
```
Comment (yury-s) [inline packages/playwright-core/src/server/deviceDescriptorsSource.json:669]: Can you point us to the place where you got this numbers? Looking at the [official site](https://www.apple.com/iphone-12/specs/#:~:text=The%20iPhone%2012%20mini%20display,actual%20viewable%20area%20is%20less) I'd expect the screen dimensions to be 360x780.

File: packages/playwright-core/src/server/deviceDescriptorsSource.json [hunk 2]
```diff
@@ -663,6 +663,36 @@
...
+      "height": 812
+    },
+    "viewport": {
+      "width": 375,
+      "height": 629
```
Comment (mxschmitt) [inline packages/playwright-core/src/server/deviceDescriptorsSource.json:674]: where did you get the 629 from? From my experiments I get 743. The other numbers sound reasonable to me.

File: packages/playwright-core/src/server/deviceDescriptorsSource.json (hunk 1 above)
Comment (mxschmitt) [reply packages/playwright-core/src/server/deviceDescriptorsSource.json:669]: I double checked with an emulator, lgtm except the question in https://github.com/microsoft/playwright/pull/11429/files#r799014128.

File: packages/playwright-core/src/server/deviceDescriptorsSource.json [hunk 3]
```diff
@@ -663,6 +663,36 @@
...
+  },
+  "iPhone 12 Mini landscape": {
+    "userAgent": "Mozilla/5.0 (iPhone; CPU iPhone OS 14_4 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/15.4 Mobile/15E148 Safari/604.1",
+    "screen": {
+      "width": 375,
+      "height": 812
+    },
+    "viewport": {
+      "width": 712,
+      "height": 327
```
Comment (mxschmitt) [inline packages/playwright-core/src/server/deviceDescriptorsSource.json:689]:
```suggestion
      "height": 325
```

**Review by mxschmitt — APPROVED** [review]

**Pull request merged**